js/ad_manager_vast.js

```diff
@@ -837,8 +1176,8 @@ OO.Ads.manager(function(_, $) {
      * The xml needs to be parsed to grab all the linear data of the ad and create an object.
      * @private
      * @method Vast#parseLinearAd
-     * @param {xml} Xml containing the ad data to be parsed.
-     * @returns {object} result An object containing the ad data.
+     * @param {xml} linearXml The xml containing the ad data to be parsed
```

Comment:
you're right, there isn't an xml primitive. I'll make the change for the rest of them